### PR TITLE
fix(jellyfish-api-core): Fix failed tests after connecting to the latest ain code

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What this PR does / why we need it:
Fix failed tests after connecting to the latest ain code.
See https://github.com/DeFiCh/ain/pull/1320

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/vault/listAuctionHistory.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import BigNumber from 'bignumber.js'
 import { TestingGroup } from '@defichain/jellyfish-testing'
 import { GenesisKeys, MasterNodeRegTestContainer } from '@defichain/testcontainers'
-import { ListAuctionHistoryDetail, VaultLiquidation } from '../../../src/category/vault'
+import { ListAuctionHistoryDetail, VaultLiquidation } from '../../../src/category/loan'
```
Comment (ivan-zynesis) [inline packages/jellyfish-api-core/__tests__/category/vault/listAuctionHistory.test.ts:4]: `vault` is refactored version, revert this line, test should remained working.

Comment (ivan-zynesis) [inline packages/jellyfish-api-core/__tests__/category/vault/listAuctionHistory.test.ts:4]:
```suggestion
import { ListAuctionHistoryDetail, VaultLiquidation } from '../../../src/category/vault'
```

**Review by ivan-zynesis — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/loan/listAuctionHistory.test.ts [hunk 2]
```diff
@@ -615,25 +615,25 @@ describe('Loan listAuctionHistory', () => {
           { index: 1 }
         )
 
-        expect(page.length).toStrictEqual(4)
+        expect(page.length).toStrictEqual(2)
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/listAuctionHistory.test.ts:618]:
```
// This test shows that we can not use index only for the pagination start point as the DB composite index starts with maxBlockHeight.
```
so earlier, it did not work with `index` only and now it works? if then the comment should be removed.
Wondering what changed it. likewise does "should not listAuctionHistory with vaultId only" also changed?

**Comment by surangap** [comment]
> Since it's related to product APIs, I need to understand the context of this change. What caused the size of the auction history to change in this release?

yes, something is not right. let me check

**Comment by surangap** [comment]
> > Since it's related to product APIs, I need to understand the context of this change. What caused the size of the auction history to change in this release?
> 
> yes, something is not right. let me check cpp side changes



> > Since it's related to product APIs, I need to understand the context of this change. What caused the size of the auction history to change in this release?
> 
> yes, something is not right. let me check cpp side changes

https://github.com/DeFiCh/ain/pull/1215/files
Looks like this implements a filter for `BatchIndex` which is not what we want. what we need is pagination, not filtration.

**Pull request closed without merging**